releases/client/2.4/settings

```diff
@@ -0,0 +1,3 @@
+FOREMAN_VERSION="2.4"
+FULLVERSION=$FOREMAN_VERSION
+OSES="rhel6 rhel7 el8 sles11 sles12"
```

Comment:
Will we keep building for RHEL6? CentOS 6 is EOL now.

Reply:
Given our downstream still supports it, I'd prefer to keep it.